policyengine_canada/variables/gov/provinces/qc/tax/income/credits/sa/qc_sa_tax_credit.py

```diff
@@ -0,0 +1,37 @@
+from policyengine_canada.model_api import *
+
+
+class qc_sa_tax_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec senior assistance tax credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.sa
+
+        income = household("adjusted_family_net_income", period)
+
+        # Your spouse is an eligible individual and you were both 70 or over
+        married_both_eligible = household(
+            "qc_sa_married_both_eligible", period
+        )
+        credit1 = married_both_eligible * max_(
```

Comment:
Verbose but preferable if it avoids using numbered variables (unless they're numbered in the law or tax forms)
```suggestion
        credit_married_both_eligible = married_both_eligible * max_(
```